scripts/split-e2e-tests.ts

```diff
@@ -474,29 +382,29 @@ function main(): void {
   const splitPkgTests = splitTests(
     config,
     'amplify_e2e_tests_pkg',
-    'build_test_deploy',
+    'build_test_deploy_v3',
```

Comment:
Should this be changed back before merging to dev?

Reply:
I did this intentionally, since our current build_test_deploy job is cluttered & freezes up due to the number of old job names that are associated with it. Github & other branch rules will automatically apply (per Ed's comments, and I've verified that checks are automatically picked up here:
![image](https://user-images.githubusercontent.com/110861985/203190848-a4eb56e4-a95c-459a-9957-6fae443e8457.png)